tbx/work/models.py

```diff
@@ -222,6 +245,7 @@ class WorkPage(ColourThemeMixin, SocialFields, Page):
     promote_panels = [
         MultiFieldPanel(Page.promote_panels, "Common page configuration"),
         FieldPanel("listing_summary"),
+        FieldPanel("related_sectors", widget=forms.CheckboxSelectMultiple),
         FieldPanel("related_services", widget=forms.CheckboxSelectMultiple),
```

Comment:
**Suggestion**

To combine these using a `MultiFieldPanel` as done here:

https://github.com/torchbox/torchbox.com/blob/ff717ca065cfc01e146e8c2d16c0e025aa740cf4/tbx/blog/models.py#L225-L231